static/app/components/modals/sudoModal.tsx

```diff
@@ -177,7 +158,7 @@ class SudoModal extends React.Component<Props, State> {
               initialData={{isSuperuserModal: isSuperuser}}
               resetOnError
             >
-              {!isSelfHosted && showAccessForms && isSuperuser && (
+              {!isSelfHosted && isSuperuser && (
```

Comment:
This `isSuperuser` is not needed because it's already checked in Line 151.

Reply:
^double this.